### PR TITLE
add link to flutter tool extension API design doc

### PR DESCRIPTION
**Pull request by christopherfujino**
Add link to https://docs.google.com/document/d/12iXm23TP6e6qS1Ntx-i-azcBWByiUl_kWjRxRC2YTz8/edit?usp=sharing design doc.

### CODE REVIEW

**Review by parlough — CHANGES_REQUESTED** [review]
Thanks!

I can't seem to access the doc whether logged in or not though, can you either share it or use a link with a resource key if it's already shared?

**Comment by christopherfujino** [comment]
> Thanks!
> 
> I can't seem to access the doc whether logged in or not though, can you either share it or use a link with a resource key if it's already shared?

Hmm, the permissions on the doc says anyone with the link can comment...

**Comment by parlough** [comment]
Does the link in the share window have a resource key?

File: firebase.json [hunk 1]
```diff
@@ -301,6 +301,7 @@
       { "source": "/go/flutter-plugins-repo-migration", "destination": "https://docs.google.com/document/d/1VihAgx1e-X_H8VRc9WmUv_7qqsi81eeHrrhwhHgwtjQ/edit?usp=sharing", "type": 301 },
       { "source": "/go/flutter-skia-ios-font-problem", "destination": "https://docs.google.com/document/d/1UkAhEKUEWNUjfKlqoBjJnx8zXrqErIqidPYjd9FBwsg", "type": 301 },
       { "source": "/go/flutter-support-multi-architecture", "destination": "https://docs.google.com/document/d/19tzWySgtgtTA99XQsjx5Pg0SFJeZKXyUlYavR0EXv8c/edit?usp=sharing", "type": 301 },
+      { "source": "/go/flutter-tool-extension-api", "destination": "https://docs.google.com/document/d/12iXm23TP6e6qS1Ntx-i-azcBWByiUl_kWjRxRC2YTz8/edit?usp=sharing", "type": 301 },
```
Comment (parlough) [inline firebase.json:304]: I was able to access it with a different account. This link works on incognito too:

```suggestion
      { "source": "/go/flutter-tool-extension-api", "destination": "https://docs.google.com/document/d/12iXm23TP6e6qS1Ntx-i-azcBWByiUl_kWjRxRC2YTz8/edit?usp=sharing&resourcekey=0-pVl2giJzGAHqOYEaZagVow", "type": 301 },
```

**Comment by christopherfujino** [comment]
Good point! What about: https://docs.google.com/document/d/12iXm23TP6e6qS1Ntx-i-azcBWByiUl_kWjRxRC2YTz8/edit?usp=sharing&resourcekey=0-pVl2giJzGAHqOYEaZagVow

@parlough

if that works I can update my PR

**Review by parlough — APPROVED** [review]
Thanks!

**Comment by parlough** [comment]
Oh sorry, didn't see your comment. I pushed a commit adding the resource key :)

**Pull request merged**